package.json

```diff
@@ -26,8 +27,8 @@
     "NavigationTestUtils.js"
   ],
   "peerDependencies": {
-    "react": "*",
-    "react-native": "*"
+    "react": "16.13.1",
```

Comment:
let's revert this back to *, we don't need a specific version in here I think.

Reply:
Done in the latest commit